packages/project-access/src/project/info.ts

```diff
@@ -0,0 +1,33 @@
+import type { Editor } from 'mem-fs-editor';
+import { join } from 'path';
+import { FileName } from '../constants';
+import { fileExists, findFilesByExtension } from '../file';
+import { getWebappPath } from './ui5-config';
+
+/**
+ * Get the used programming language of an application.
+ *
+ * @param appRoot - root folder of the application
+ * @param [memFs] - optional mem-fs editor instance
+ * @returns - used language, JavaScript or TypeScript
+ */
+export async function getAppLanguage(appRoot: string, memFs?: Editor): Promise<'JavaScript' | 'TypeScript' | ''> {
+    const ignoreFolders = ['node_modules', '.git'];
+    let appLanguage: 'JavaScript' | 'TypeScript' | '' = '';
```

Comment:
Can we use and export this as a type? I am sure we will need to use something similar in the consumed tools code.